src/server/ServerHandler.java

```diff
@@ -308,6 +308,48 @@ private PairingResponse handleUpdatePairing() {
         }
     }
 
+    /**
+     * Handles the acceptance of a pending game invitation by the opponent.
+     * This method retrieves the event associated with the given {@code eventId}
+     *
+     * @param eventId the unique identifier of the event (invitation) to be accepted
+     * @return a {@link Response} object indicating whether the acceptance was successful or failed,
+     * with an appropriate message
+     */
+
+    public Response handleAcceptInvitation(int eventId) throws SQLException {
+        // Retrieve the event from the database
+        Event event = DatabaseHelper.getInstance().getEvent(currentEventId);
```

Comment:
Need to call this with `eventId` which was part of the request
```suggestion
        Event event = DatabaseHelper.getInstance().getEvent(eventId);
```